specs/Deployment/DeploymentAPI.md

```diff
@@ -155,6 +202,15 @@ namespace Microsoft.Windows.ApplicationModel.WindowsAppRuntime
         HRESULT ExtendedError{ get; };
     };
 
+    /// This object is used to specify deployment options to apply when using DeploymentManager's
+    /// Initialize method
+    runtimeClass DeploymentOptions
```

Comment:
This will need to declare an APIcontract. It adds an interface in 1.1 to a runtimeclass created in 1.0

The fine print's being defined. @DefaultRyan and @dhoehna are working on spec/docs for that. For now you can add

```
//TODO:12345 add APIcontract once WinAppSDK's rules for them are defined [contract(name,version)]
```

as a marker where the contract's needed. Also create an ADO task to track that